src/style.css

```
@@ -0,0 +1,120 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 500;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+body {
```

```suggestion
html {
```
fonts should be set in `html` selector tag